.github/workflows/node-latest.yml

```diff
@@ -1,5 +1,6 @@
 name: Get latest Node release version
 on:
+  workflow_dispatch:
   schedule:
     - cron:  '* * */5 * * *'
```

Comment:
This may conflict again. Maybe the alpha branch is not being pulled into the fork prior to submitting the next PR? This was resolved with an upstream merge. 

I find it much easier to rebase a branch inside guild-operators spawned from origin/alpha than it is to rebase my own fork of origin/alpha. Might make things simpler to change to using a branch as a collaborator for PR's vs. a repo fork as a community submission for PR's.